minimal/server.go

```
@@ -190,6 +191,24 @@ func (t *txpoolHub) GetNonce(root types.Hash, addr types.Address) uint64 {
 	return account.Nonce
 }
 
+func (t *txpoolHub) GetBalance(root types.Hash, addr types.Address) *big.Int {
+	zeroValue := big.NewInt(0)
+
+	snap, err := t.state.NewSnapshotAt(root)
+	if err != nil {
+		return zeroValue
```

I think we should return an error here if we cannot find snapshot at all.

---

Fixed 👍 

2771012